packages/rocketchat-ui-flextab/flex-tab/tabs/uploadedFilesList.coffee

```diff
@@ -2,7 +2,7 @@ roomFiles = new Mongo.Collection 'room_files'
 
 Template.uploadedFilesList.helpers
 	files: ->
-		return roomFiles.find({ rid: @rid }).fetch()
+		return roomFiles.find({ rid: @rid }).fetch().sort({uploadedAt : -1})
```

Comment:
The .sort at the end only "seems" to work because it is being applied to an array (the result of .fetch()). But sorting an array of objects has not the desired effect. @rodrigok is right, this should be done inside the .find().